README.md

```diff
@@ -1,3 +1,21 @@
+# ** Announcement ** - Nancy is no longer being maintained! 
+
+We would like to thank all the thousands of users of Nancy, all the people who wrote blog posts, conference speakers, video producers and those that spread the word of Nancy.  
+
+We would like to thank the 150+ contributors to Nancy that made it what it became, without you the work would have been much harder and opportunities missed.  
+
+We would like to thank [VQ](http://www.vqcomms.com) for being one of the few companies that paid for our open source efforts.  
```

Comment:
This feel like a wee dig to others. Perhaps this? "We would like to thank [VQ](http://www.vqcomms.com) for sponsoring for our open source efforts. "